quobyte_driver.go

```diff
@@ -76,12 +90,19 @@ func (driver quobyteDriver) Create(request volume.Request) volume.Response {
 		}
 	}
 
-	mPoint := filepath.Join(driver.quobyteMount, request.Name)
-	log.Printf("Validate mounting volume %s on %s\n", request.Name, mPoint)
+	mPoint := filepath.Join(driver.quobyteMount, strippedName)
+	log.Printf("Validate mounting volume %s on %s\n", strippedName, mPoint)
 	if err := driver.checkMountPoint(mPoint); err != nil {
 		return volume.Response{Err: err.Error()}
 	}
 
+	if subDirs != "" {
+		log.Printf("Creating subdir(s) %s for new volume %s\n", subDirs, strippedName)
+		if csdErr := os.MkdirAll(filepath.Join(mPoint, subDirs), 0755); csdErr != nil {
```

Comment:
Do we want to return an error or only print this information ?

Reply:
As the volume has been created i'd stick with the message at this point (instead of adding a volume deletion and error throwing).

Reply:
👍